CHANGELOG.md

```diff
@@ -1,264 +1,272 @@
 
 Features:
 
-- Dynamic workspace symbol picker (#5055)
-- Soft-wrap (#5420, #5786, #5893, #6142, #6440)
-- Initial support for LSP snippet completions (#5864, b1f7528, #6263, bbf4800, 90348b8, f87299f, #6371)
-- Add a statusline element for showing the current version control HEAD (#5682)
-- Display LSP type hints (#5420, #5934, #6312)
-- Enable the Kitty keyboard protocol on terminals with support (#4939, #6170, #6194)
-- Add a statusline element for the basename of the current file (#5318)
-- Add substring matching syntax for the picker (#5658)
-- Support LSP `textDocument/prepareRename` (#6103)
-- Allow multiple runtime directories with priorities (#5411)
-- Allow configuring whether to insert or replace completions (#5728)
+- Dynamic workspace symbol picker ([#5055](https://github.com/helix-editor/helix/pull/5055))
+- Soft-wrap ([#5420](https://github.com/helix-editor/helix/pull/5420), [#5786](https://github.com/helix-editor/helix/pull/5786), [#5893](https://github.com/helix-editor/helix/pull/5893), [#6142](https://github.com/helix-editor/helix/pull/6142), [#6440](https://github.com/helix-editor/helix/pull/6440))
+- Initial support for LSP snippet completions ([#5864](https://github.com/helix-editor/helix/pull/5864), [b1f7528](https://github.com/helix-editor/helix/commit/b1f7528), [#6263](https://github.com/helix-editor/helix/pull/6263), [bbf4800](https://github.com/helix-editor/helix/commit/bbf4800), [90348b8](https://github.com/helix-editor/helix/commit/90348b8), [f87299f](https://github.com/helix-editor/helix/commit/f87299f), [#6371](https://github.com/helix-editor/helix/pull/6371), [9fe3adc](https://github.com/helix-editor/helix/commit/9fe3adc))
+- Add a statusline element for showing the current version control HEAD ([#5682](https://github.com/helix-editor/helix/pull/5682))
+- Display LSP type hints ([#5420](https://github.com/helix-editor/helix/pull/5420), [#5934](https://github.com/helix-editor/helix/pull/5934), [#6312](https://github.com/helix-editor/helix/pull/6312))
+- Enable the Kitty keyboard protocol on terminals with support ([#4939](https://github.com/helix-editor/helix/pull/4939), [#6170](https://github.com/helix-editor/helix/pull/6170), [#6194](https://github.com/helix-editor/helix/pull/6194), [#6438](https://github.com/helix-editor/helix/pull/6438))
+- Add a statusline element for the basename of the current file ([#5318](https://github.com/helix-editor/helix/pull/5318))
+- Add substring matching syntax for the picker ([#5658](https://github.com/helix-editor/helix/pull/5658))
+- Support LSP `textDocument/prepareRename` ([#6103](https://github.com/helix-editor/helix/pull/6103))
+- Allow multiple runtime directories with priorities ([#5411](https://github.com/helix-editor/helix/pull/5411))
+- Allow configuring whether to insert or replace completions ([#5728](https://github.com/helix-editor/helix/pull/5728))
+- Add proper support for LSP workspaces ([#5748](https://github.com/helix-editor/helix/pull/5748))
```

Comment:
I wonder if we should split this up a bit because this PR contained multiple features that users would care about individually (while the title doesn't seem as useful):
* Feature: Allow per workspace config file `.helix/config.toml`
* Command: Add command to open workspace config file `:config-open-workspace`
* Breaking: Workspace detection now stops at the first `.helix` (so merging multiple `.helix/languages.toml` is not supported anymore)
* Feature: Add `workspace-lsp-roots` config option to support multiple LSP roots for use with monorepos

It would sort of overrepresent the PR but these are all separate changes from a users perspective IMO.